src/screens/CreatePost/index.js

```diff
@@ -5,6 +5,7 @@
 import * as React from 'react';
 import PSL from 'psl';
 import Toast from 'react-native-simple-toast';
+import {Image} from 'react-native-compressor';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Image` from 'react-native-compressor' has been removed. Ensure that this is not used anywhere else in the file or it will cause a reference error.
```diff
- import {Image} from 'react-native-compressor';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->